regulations/static/regulations/css/less/variables.less

```diff
@@ -33,6 +33,8 @@ variables.less contains all theme variable / variable overrides
 @red_orange_80: #D96A4F;
 @red_orange_50: #E9A193;
 
+@history_drawer_border_color: #d1d3d5;
```

Comment:
I was planning on abandoning the all caps for hex colors, but whatever the consensus is - @xtine do you prefer all caps for hex colors or mixed case?  I find the mixed a little easier to read and less shouty.

That hex color refers to a very on-off color - the line that appears above "Compare with" in the history panel 

<img width="226" alt="screen shot 2016-02-11 at 9 35 02 am" src="https://cloud.githubusercontent.com/assets/737400/12980633/ea3e0586-d0a2-11e5-9782-7c424b049b7b.png">

We changed it from light gray because the gray was barely visible.  I agree that it is a very specific naming, and probably not the best name - but it also doesn't seem to be a recurring element within the style where I can give it a more meaningful name.  Maybe @history_drawer_accent_color_secondary?